GcmEndpoints/README.md

```diff
@@ -7,7 +7,9 @@ This backend template employs [Google Cloud Endpoints](https://developers.google
 
 # 1. Adding a backend in Android Studio
 
-To add the backend to your existing Android app from this backend template, open Android Studio ([installation instructions](https://developer.android.com/sdk/installing/studio.html)) and navigate to "File &rarr; New Module..." or right-click on your project and choose "New &rarr; Module".
+Start with an existing Android app. You can use the [GCM sample Quickstart app](https://github.com/googlesamples/google-services/tree/master/android/gcm) as a starting point.
+
```

Comment:
2.3 was probably not the right jump point actually. This also depends on the goal of this guide. I was going with the assumption that this was a guide for setting up GCM, not for deploying an already set up GCM app.